README.md

```diff
@@ -105,10 +105,16 @@ quality checks and releases.
    being merged into the main branch.
 
 2. **Release Workflow**
-   When changes are pushed to the `main` branch, this workflow handles versioning and releases using
+   When changes are pushed to `main`, this workflow handles versioning and releases with
    `semantic-release`. It automatically bumps the version, generates release notes, creates a tag,
-   and publishes a Docker image to an Amazon ECR repository.
-
+   and publishes a Docker image to Amazon ECR. This new version is automatically deployed to the
+   development environment.
+
+3. **Production deployment**
+   To deploy a specific version to production, manually trigger the `Promote Version to Stable`
```

Comment:
Maybe add a link to the workflow?
```suggestion
   To deploy a specific version to production, manually trigger the [`Promote Version to Stable`](https://github.com/SRGSSR/pillarbox-event-dispatcher/actions/workflows/deploy.yml)
```

Reply:
I will maybe change this workflow later to use the prerelease/latest mechanism so I don't want to delve too much into this documentation now.